latex_access/polish_braille.py

```diff
@@ -0,0 +1,391 @@
+            return lettersign
+# Last character was part of a number and letters are within range a-j 
+        if self.lastnumber >= 0 and start == self.lastnumber and input[start].lower() in 'abcdefghij':
+            lettersign = True
+
+        try: # white space on either side of single char 
+            if start > 0 and input[start-1] == ' ' and input[start+1] == ' ':
+                lettersign = True
+        except:
+            pass
+        try: # char on it's own at beginning of line 
+            if start == 0 and input[start+1] == ' ':
+                lettersign = True
+        except:
+            pass
+        try: # char on it's own at end of line 
+            if input[start-1] in ' .,$' and start+1 == len(input):
+                lettersign = True
+        except:
+            pass
+        try: # some punctuation after a letter eg. A.
+            if input[start+1] in '.,$' and not input[start-1].isalpha ():
+                lettersign = True
+        except:
+            pass
+
+        try: # Letters on the left punctuation, on the right punctuation or space 
+            if input[start-1] in '.,$}' and not input[start+1].isalpha ():
+                lettersign = True
+        except:
+            pass
+
+        try: # Letters on the left punctuation, on the right punctuation or space 
+            if input[start-1] in '}':
+                lettersign = True
+        except:
+            pass
+            
+        if len (input) == 1: # char by itself on line
+            lettersign = True
+        return lettersign 
+
+    def lowerLetter (self,input,start):
+        '''Translates lower case letters in latex.
+
+        Returns a touple as above.'''
+        start=start-1 # We are manipulating current char
+        if self.letterSign(input,start): # add the letter sign 
+          translation = u"⠠"
+        else:
+            translation = ''
+        if start > 1 and input[start-2:start].isupper () and input[start-2:start].isalpha () and input[start].islower (): # capital letters and now we are a lower case 
+            translation+=u"⠠"
+        translation += input[start]
+        return (translation,start+1)
+
+    def upperLetter (self, input, start):
+        '''Translates upper case letters in latex.
+
+        Returns a touple as above.'''
+        if self.capitalisation == '8dot':
+            return (input[start-1], start)
+        start=start-1 # We are focused on current char 
+        translation= "" # The brf translation 
+        cap = True # Provide a capital sign unless special case (below)
+        doublecap = False # Do we represent consecutive capital letters by ,,
```

Comment:
Please fix this comment so that it mentions the correct symbol for consecutive capitals for Polish Braille (I believe ",," is used in UEB, as that is where we copied this code from in 2018:-)